lib/validators/ajv-validator.ts

```diff
@@ -14,17 +15,47 @@ export class AjvValidator implements IValidator {
     this._ajv = new Ajv(options);
   }
 
-  public loadSchemas(schemas: any[]): void {
+  public loadSwaggerSchemas(schemas: SchemaObject[]): void {
     schemas.forEach((schema) => {
+      schema = this.addPrefixToRefs(VALIDATOR_ID_PREFIX, schema);
+
+      if (schema.$id) {
+        schema.$id = `${VALIDATOR_ID_PREFIX}${schema.$id}`;
+      }
+
+      if (schema.required === true) {
```

Comment:
а тебе прям проверки на true надо делать или можно на truly? То есть может вот так будет лучше?

```suggestion
      if (schema.required) {
```

Reply:
там именно true, потому что может быть массив и он тоже является == true